### ISSUE TITLE
Check off assignments individually

### ISSUE THREAD

**Issue by willbarkoff** [PlanHubMe/PlanHub#248]
The user should be able to check off assignments individually, where it will just cross off the assignment. When all assignments for a certain subject are completed, the box should shade. This is some `javascript` and some `css`.

**Comment by thatoddmailbox** [comment]
@nywillb I don't know how easy/possible it is to strikeout part of a textarea in CSS..

**Comment by amazansky** [comment]
couldnt we just add span id="..." around each event the user makes with the id being like the title of the event or something and then make checkboxes after each event?

**Comment by willbarkoff** [comment]
@thatoddmailbox or we could add div tags with classes or ids around the individual assignments. I don't know much js so I don't know how hard that would be, but just thinking about it, the doesn't seem that hard.

**Comment by thatoddmailbox** [comment]
@RobotXLabs @nywillb no, because textareas' values aren't made up of HTML tags.

This does _not_ work:

``` html
<!-- doesn't work -->
<textarea>
    <span id="thingy">asdf</span>
    <span id="otherThingy">qwer</span>
</textarea>
```

The textarea will end up saying "<span id="thingy">asdf</span><span id="otherThingy">qwer</span>" (directly as that, showing the HTML tags as text).

**Comment by amazansky** [comment]
`<br \>`, no?
_edited by @nywillb for better syntax_

**Comment by thatoddmailbox** [comment]
@RobotXLabs no `<br \>` in textareas, you have to use newlines (JavaScript uses `"\n"`)

**Comment by amazansky** [comment]
Or that works too

**Comment by thatoddmailbox** [comment]
@RobotXLabs no, the problem is not that there are no newlines, but that there are no ways to put HTML tags in `textarea`s, so you can't do partial strikeouts.

**Comment by amazansky** [comment]
Is this done with the commit that allows you to check off from hwview?

**Comment by thatoddmailbox** [comment]
@RobotXLabs No, this is to allow you to check off assignments in the planner individually.
